### ISSUE TITLE
iphone and ipad

### ISSUE THREAD

**Issue by RFCBox** [EricssonResearch/openwebrtc#570]
i'm a chinese starting coder ,my english is not good.  i run the demo on iphone well. but when i run it on the ipad,it's very very slowly (establishing  connection). i don't know where is wrong!

**Comment by stefhak** [comment]
Are the iPhone and iPad on the same (WiFi) network, or on different
networks?

**Comment by RFCBox** [comment]
on the same network, when i run the demo on the iphone ,the demo works well. but on the ipad ,I can only see the local video stream, and can't see the remote video stream.Or that the remote video stream loads very slowly.

**Comment by stefhak** [comment]
Just to establish some date: - you use Bowser I assume? Which version of Bowser? What version of iOS on the devices.
Also, @stefanalund @superdump do you know a reason to expect this? Should this Issue be closed and a new one be opened for Bowser instead?

**Comment by stefanalund** [comment]
Yep, we need a bit more information about which clients you are using. As @stefhak said, if it is Bowser then it is better to open an issue in that repository: https://github.com/EricssonResearch/bowser

**Comment by RFCBox** [comment]
@stefanalund ,i run demo on iphone6 and ipad2.

**Comment by RFCBox** [comment]
@stefanalund @superdump 
i just run it on the ipadmini,it run well yet.just ipad2 run faild.    there is some information in xcode:

==== Warning message start ====
Warning in element audio-source.
Warning: Can't record audio fast enough
Debugging info: gstaudiobasesrc.c(866): GstFlowReturn gst_audio_base_src_create(GstBaseSrc *, guint64, guint, GstBuffer **) (): /GstPipeline:local-audio-capture-source-bin-0/GstOsxAudioSrc:audio-source:
Dropped 1764 samples. This is most likely because downstream can't keep up and is consuming samples too slowly.
==== Warning message stop ====

**Comment by stefanalund** [comment]
@RFCBox we still need to know which client you are testing with (Bowser, NativeDemo, ...).

**Comment by RFCBox** [comment]
@stefanalund ,i don't know your true meaning,i'm sorry!
iphone6 :  ios9.2  +   bowser : 0.6.1  =   well ;
ipad mini : ios8.3  +  bowser : 0.6.1 = well;
ipad2 : ios9.2  +  bowser  :  0.6.1  = not  well.     the warning message is  above.


I don't know whether it is a hardware problem, or the reason why Bowser has not been handled internally.

**Comment by tobi512** [comment]
@RFCBox The performance of an iPad 2 is just not good enough for proper streaming in this case, especially compared to iPhone 6 and iPad mini.
Even when I developed with an iPad 3 the same warnings come up. You can try to reduce audio/video bandwidth, but the device is probably simply too slow...
